toolkits/springernature/packages/springernature-forms/scss/50-components/_forms.scss

```diff
@@ -0,0 +1,121 @@
+.c-form__section {
+	margin-bottom: $forms--section-margin-bottom;
+}
+
+.c-form__field {
+	width: 100%;
+
+	margin-bottom: $forms--field-margin-bottom;
+	line-height: 1.2;
+	font-size: $forms--font-size;
+	font-family: $context--font-family-sans;
+
+	fieldset {
+		margin: 0;
+		padding: 0;
+		border: 0;
+	}
+}
+
+@include from-wide-tablet {
+	.c-form__field {
+		max-width: $forms--input-max-width;
+	}
+}
+
+.c-form__label-text {
+	display: block;
+	font-weight: $context--font-weight-bold;
+	margin-bottom: $forms--label-text-margin-bottom;
+}
+
+.c-form__hint {
+	display: block;
+	margin-bottom: $forms--hint-margin-bottom;
+	color: $forms--hint-color;
+}
+
+.c-form__input {
+	@include forms--input;
+}
+
+.c-form__heading {
+	margin-bottom: $forms--heading-margin-bottom;
+}
+
+.c-form__subheading {
+	margin-bottom: $forms--subheading-margin-bottom;
+}
+
+.c-form__radio {
+	display: block;
+	padding: $forms--radio-padding;
+}
+
+.c-form__radio [type="radio"] {
+	margin-right: $forms--radio-margin-right;
+}
+
+.c-form__checkbox {
+	display: block;
+	padding: $forms--checkbox-padding;
+}
+
+.c-form__checkbox [type="checkbox"] {
+	margin-right: $forms--checkbox-margin-right;
+}
+
+.c-form__select {
+	@include forms--input;
+	padding: 0;
+	position: relative;
+
+	&:after {
+		content: '';
+		position: absolute;
+		top: .8em;
+		right: .5em;
+		border-left: .4em solid transparent;
+		border-right: .4em solid transparent;
+		border-top: .5em solid $forms--select-arrow-color;
+		display: block;
+	}
+
+	& select {
+		appearance: none;
+		line-height: 1.4;
+		background-color: transparent;
+		border: none;
+		padding: $forms--select-padding;
+		display: block;
+		width: 100%;
+
+		&::-ms-expand {
+			display: none;
+		}
+	}
+}
+
+.c-form__select
+.c-form__label-text--inline {
```

Comment:
why is this needed? the `display: inline` should just be included in `c-forms__select` with the rest of the styles, duplication is better here than saving one line.

why is `c-form__label-text--inline` needed?

Reply:
> duplication is better here than saving one line
💯  will fix that

`c-form__label-text--inline` if for when a label needs to be inlined with a checkbox or radio for instance

Reply:
Oops fixed this as part of https://github.com/springernature/frontend-toolkits/pull/518/commits/1390aa0eb1d61d873e2b5ea40643b5b547957a2f 🤦, but it's in there now

Reply:
I got confused, this was a mistake though. I think it read as if there was a comma after `.c-form__select` therefore the remark with the display inline for that selector too. I started putting the inline on the `.c-form__select` and screwed my dropdown. Took me quite some inspection to find out that `display: inline` was messing it up because of overriding the display block. Now this looks as it should look :)